### ISSUE TITLE
Migrate Garvin City Planning Image Collection

### ISSUE THREAD

**Issue by motropuk** [yalelibrary/YUL-DC#1807]

**Comment by MaggieZhaoYale** [comment]
over 250,000 images. Downloaded 9999 each run.

**Comment by MaggieZhaoYale** [comment]
re-open service now ticket #2117131 increase space on the image shares. @bobrice

**Comment by motropuk** [comment]
@MaggieZhaoYale Spotted some issues during auditing on UAT. Parent 16195742 does not have any child objects attached. Additionally, while there are 16557 objects in DCS UAT management app, only 15364 are displaying in blacklight https://collections-uat.library.yale.edu/catalog?f%5bproject_identifier_tesi%5d%5b%5d=65&q=&search_field=all_fields. 1 parent (16195742) is Yale Community Only so that would account for 1 parent not displaying. But that still leaves 1192 parents that are not displaying in Blacklight

**Comment by MaggieZhaoYale** [comment]
Reprocess the failed parents. 887 parents failed. Among them, 156 are the parents without images.
Some failed parents have children which only have jpg images. e.g. digcoll:4228020. Shall we use the jpg images and converted them to tif? @motropuk 
The list of parent without images in the failed parents. 
[https://app.zenhub.com/files/242185115/b0f3b5f7-5dc5-4745-ab1e-a21018d33546/download](https://app.zenhub.com/files/242185115/b0f3b5f7-5dc5-4745-ab1e-a21018d33546/download)

**Comment by motropuk** [comment]
@MaggieZhaoYale Yes lets cover the ones with jpgs only to TIFF and then lets see what we have left after that.

**Comment by motropuk** [comment]
Still only 15,660 objects out of the 16559 in UAT management, displaying in UAT blacklight  https://collections-uat.library.yale.edu/catalog?f%5bproject_identifier_tesi%5d%5b%5d=65&q=&search_field=all_fields. @MaggieZhaoYale is it possible to get a full list of the parents that are not displaying on the front end?

**Comment by MaggieZhaoYale** [comment]
Failed to convert the jpg to tif using imageMagik. Still 15660/16559 in blacklight.
Among the 899 problematic parents, 166 are without images

**Comment by motropuk** [comment]
@MaggieZhaoYale We are going to take the same approach with this collection as we did with Day Missions. Lets move everything to Production and then we will audit and fix issues there. Checked in with Arts and they are happy with this approach

**Comment by MaggieZhaoYale** [comment]
@motropuk @sshetenhelm 
851 parent failed in the prod.
Among 851 failed parent, 234 are inactive parents or deleted parents or parent without images.
Here is the list of 234 parents:
[https://app.zenhub.com/files/242185115/3e928933-2f39-4424-a643-508362440e13/download](https://app.zenhub.com/files/242185115/3e928933-2f39-4424-a643-508362440e13/download)
The causes of 234 deleted/inactive parents or parent without images: Fedora query the active children with viewopt_ssi archivalDigitized, then get their parents. So parent could be deleted or inactive in the fedora. some parent without children came from ladybird information.

Others are parents with missing images. Except couple of the parents have one image could not convert into ptiff (images issue), most of them of them are caused by: findit has less children than that in the ladybird. e.g. 16190193 has 19 images, ladybird has 22 images. 
Here is the list of the parents with missing/problematic   images: 
[https://app.zenhub.com/files/242185115/04a0b97c-7d66-46e9-adc6-37dc0eaa863e/download](https://app.zenhub.com/files/242185115/04a0b97c-7d66-46e9-adc6-37dc0eaa863e/download)

**Comment by motropuk** [comment]
Thanks @MaggieZhaoYale. I will link to your note above in our review ticket for Garvin (#1929). So all remaining work for Garvin is in #1929. This ticket can be closed. Confirmed that 16559 objects are in Production management app.
